Apps/W1/EDocumentsConnector/app/src/Continia/Metadata/Network.Enum.al

```diff
@@ -0,0 +1,19 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+
+enum 6390 "Network"
```

Comment:
This is a very global enum name, perhaps something more related?

Reply:
We have renamed this enum to "E-Delivery Network" as this describes the Enum much better.